sympy/sets/__init__.py

```diff
@@ -3,3 +3,7 @@
 from .fancysets import ImageSet, Range, ComplexRegion
 from .contains import Contains
 from .conditionset import ConditionSet
+
+from ..core.singleton import S
```

Comment:
[This question](http://stackoverflow.com/questions/72852/how-to-do-relative-imports-in-python) may be related.